plugins/inputs/http/README.md

```diff
@@ -66,14 +66,21 @@ to use them.
   # http_proxy_url = ""
 
   ## Optional TLS Config
-  # tls_ca = "/etc/telegraf/ca.pem"
-  # tls_cert = "/etc/telegraf/cert.pem"
-  # tls_key = "/etc/telegraf/key.pem"
-  ## Minimal TLS version to accept by the client
-  # tls_min_version = "TLS12"
+  ## Set to true/false to enforce TLS being enabled/disabled. If not set,
+  ## enable TLS only if any of the other options are specified.
+  # tls_enable =
+  ## Trusted root certificates for server
+  # tls_ca = "/path/to/cafile"
+  ## Used for TLS client certificate authentication
+  # tls_cert = "/path/to/certfile"
+  ## Used for TLS client certificate authentication
+  # tls_key = "/path/to/keyfile"
+  ## Send the specified TLS server name via SNI
+  # tls_server_name = "kubernetes.example.com"
   ## Use TLS but skip chain & host verification
   # insecure_skip_verify = false
 
+
```

Comment:
This extra whitespace line seems to be in every config with this. Is this an extra from the config_includer?

Reply:
I guess it is due to the template or it is in the included file. The `config_includer` itself will only render the Golang text template no extras...

Reply:
Fixed.